setup.py

```diff
@@ -9,23 +7,10 @@
 from skbuild import setup
 
 __version__ = get_version(root=".", relative_to=__file__)
-
-logging.basicConfig(level=logging.INFO)
-log = logging.getLogger("cuvec.setup")
-
 build_ver = ".".join(__version__.split('.')[:3]).split(".dev")[0]
-cmake_args = [f"-DCUVEC_BUILD_VERSION={build_ver}", f"-DPython3_ROOT_DIR={sys.prefix}"]
-try:
-    from miutil import cuinfo
-    nvcc_arch_raw = map(cuinfo.compute_capability, range(cuinfo.num_devices()))
-    nvcc_arches = {"%d%d" % i for i in nvcc_arch_raw if i >= (3, 5)}
```

Comment:
this hack is required due to https://gitlab.kitware.com/cmake/cmake/-/issues/24921

Reply:
moved to follow-up #30